Calendar/View Controllers/ComposeViewController.m

```diff
@@ -54,8 +67,21 @@ - (Event *)createEventFromView {
     } else {
         newEvent.eventDescription = self.composeView.descriptionTextView.text;
     }
+}
+
+- (BOOL)inputIsValid {
```

Comment:
Be very careful about side effects in code. Because this intentionally has a side effect, it must be named in a way that calls it out, like `validateInput` or something.